### DISCUSSION TITLE
How do we uninstall this?

### DISCUSSION THREAD

**Discussion by grexzen** [hlky/stable-diffusion-webui#945] [Q&A] [answered]
Just deleting the Miniconda env and the main files folder? Thanks!

**Comment by Heathen** [comment, marked as answer]
If you uninstall Miniconda as well, it will leave things these folders:
/user/username_here/miniconda3/
and
/user/username_here/.cache/

You might wanna delete .cache anyway after, A lot of the models will reside there.
